must-gather/cmd/main.go

```
@@ -0,0 +1,58 @@
+		fmt.Fprintf(os.Stderr, "Failed to create destination directory: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Set up logging
+	logFilePath := filepath.Join(destDir, logFileName)
+	logFile, err := os.Create(logFilePath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create log file: %v\n", err)
+		os.Exit(1)
+	}
+	defer logFile.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `logFile.Close` is not checked

(errcheck)